From a43a3327ccd4b06a3bcf0c87d518a97c6b39ac02 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@cn.fujitsu.com>
Date: Sat, 6 Aug 2016 02:09:53 +0900
Subject: [PATCH] daemontools: Fix a warning

To fix the warning as following:

WARNING: daemontools-native-0.76-r0 do_populate_sysroot: File '/build-poky/tmp/sysroots/x86_64-linux/usr/bin/chkshsgr' from daemontools-native was already stripped, this will prevent future debugging!

Signed-off-by: Lei Maohui <leimaohui@cn.fujitsu.com>
---
 src/conf-ld | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/conf-ld b/src/conf-ld
index 59a0de7..1d0518a 100644
--- a/src/conf-ld
+++ b/src/conf-ld
@@ -1,3 +1,3 @@
-gcc -s
+gcc
 
 This will be used to link .o files into an executable.
-- 
2.7.4

